hack/helm-patches/manager.yaml.patch

```diff
@@ -6,4 +6,7 @@
             - --application-images-registry={{ .Values.application.imagesRegistry }}
             {{- if .Values.accessResources.traefik.enable }}
             - --watch-traefik
+            {{- end}}
+            {{- if .Values.extensionApi.enable }}
+            - --enable-extension-api=true
```

Comment:
this should be passed as a `boolean` flag (look at `--watch-traefik` above. Please add to the GH issue that tracks the fast follows.